changelog/unreleased/pull-2546

```diff
@@ -0,0 +1,8 @@
+Change: Exit status 2 when errors occured while reading source data. Previously, 
```

Comment:
This must be a _single_ line. Currently this gets cut off in the changelog:
```
 * Chg #2482: Remove vendored dependencies
 * Chg #2546: Exit status 2 when errors occured while reading source data. Previously,
 * Chg #2600: Require Go >= 1.11
```

What about `Exit with status 2 if some source files could not be read`?